lms/djangoapps/discussion_api/serializers.py

```diff
@@ -255,12 +265,12 @@ class CommentSerializer(_ContentSerializer):
     at introspection and Comment's __getattr__.
     """
     thread_id = serializers.CharField()
-    parent_id = serializers.CharField(required=False)
+    parent_id = serializers.CharField(required=False, allow_null=True)
     endorsed = serializers.BooleanField(required=False)
-    endorsed_by = serializers.SerializerMethodField("get_endorsed_by")
-    endorsed_by_label = serializers.SerializerMethodField("get_endorsed_by_label")
-    endorsed_at = serializers.SerializerMethodField("get_endorsed_at")
-    children = serializers.SerializerMethodField("get_children")
+    endorsed_by = serializers.SerializerMethodField()
```

Comment:
DRF v3 automatically uses `get_foo` if the field is called `foo`.  It also raises an assertion error if you try to tell it to use `get_foo` and the field is already named foo.